### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by michaelSkaro**

### CODE REVIEW

**Comment by y1zhou** [comment]
Hey @michaelSkaro, sorry if I spammed your inbox. Didn't realize the bot was going to post each comment separately. I think the notebook looks good mostly, and we just need to clean it up a bit and make it more "report-like". Thanks!

**Comment by michaelSkaro** [comment]
<p>Unused cells removed</p>

---

 <a href='https://app.reviewnb.com/y1zhou/chest-xray-pneumonia-clasifier/pull/2/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by michaelSkaro** [comment]
<p>all imports are in first cell. All duplicated imports are resolved. Sorry I built this out sorta over a few weeks and did't clean it.</p>

---

 <a href='https://app.reviewnb.com/y1zhou/chest-xray-pneumonia-clasifier/pull/2/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by michaelSkaro** [comment]
<p>Got it.</p><p> </p>

---

 <a href='https://app.reviewnb.com/y1zhou/chest-xray-pneumonia-clasifier/pull/2/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by michaelSkaro** [comment]
<p>Done.</p>

---

 <a href='https://app.reviewnb.com/y1zhou/chest-xray-pneumonia-clasifier/pull/2/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by michaelSkaro** [comment]
<p>annot.json are labels converted from your code in the heart of the mode. I will make the necessary additions to the repo. The code is on the server, I can just ingest as an import, the file is uploaded as a commit</p><p> </p>

---

 <a href='https://app.reviewnb.com/y1zhou/chest-xray-pneumonia-clasifier/pull/2/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by michaelSkaro** [comment]
<p>Cut out. Done</p><p> </p>

---

 <a href='https://app.reviewnb.com/y1zhou/chest-xray-pneumonia-clasifier/pull/2/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by michaelSkaro** [comment]
Add the json and complete updates from may 8

**Comment by y1zhou** [comment]
<p>The model weights are in <code>model_weights/aug_model.pt.ckpt</code> and you can load the checkpoint following <a href="https://pytorch-lightning.readthedocs.io/en/latest/common/weights_loading.html?highlight=load%20checkpoint#checkpoint-loading" target="_blank">this tutorial</a>.</p><p> </p><pre class="ql-syntax" spellcheck="false">from core.model import PneumoniaDetector
model = PneumoniaDetector.load_from_checkpoint(
    "model_weights/aug_model.pt.ckpt"
)
model.eval()
y_hat = model(x)
</pre><p> </p><p> </p>

---

 <a href='https://app.reviewnb.com/y1zhou/chest-xray-pneumonia-clasifier/pull/2/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>
